link/uprobe_test.go

```diff
@@ -69,6 +69,30 @@ func TestExecutableOffset(t *testing.T) {
 	c.Assert(offset, qt.Equals, uint64(0x1+0x2))
 }
 
+func TestExecutableLazyLoadSymbols(t *testing.T) {
+	c := qt.New(t)
+
+	ex, err := OpenExecutable("/bin/bash")
+	c.Assert(err, qt.IsNil)
+	// Addresses must be empty, will be lazy loaded.
+	c.Assert(ex.addresses, qt.CmpEquals(), map[string]uint64{})
```

Comment:
Huh, isn't `CmpEquals()` the same as `Equals`?

Reply:
CmpEquals uses go-cmp's `.Diff()` while Equals uses `==`

Reply:
Ah! Maybe https://pkg.go.dev/github.com/frankban/quicktest@v1.14.4#DeepEquals is more idiomatic then?

Reply:
👍🏻 looks better